Engine/DataFeeds/SubscriptionUtils.cs

```diff
@@ -82,9 +92,21 @@ public static class SubscriptionUtils
                             return false;
                         }
 
-                        var subscriptionData = SubscriptionData.Create(subscription.Configuration, exchangeHours,
-                            subscription.OffsetProvider, enumerator.Current);
+                        var data = enumerator.Current;
+
+                        if (enablePriceScale && data?.Time.Date > lastTradableDate)
+                        {
+                            lastTradableDate = data.Time.Date;
+                            currentScale = GetScaleFactor(factorFile, config.DataNormalizationMode, data.Time.Date);
```

Comment:
There's a race condition between this `config.DataNormalizationMode` and the internal to `                         SubscriptionData subscriptionData = SubscriptionData.Create`, think we should get the mode once